defaults.js

```diff
@@ -4,11 +4,11 @@ module.exports = {
     '@apostrophecms/util': {},
     '@apostrophecms/task': {},
     '@apostrophecms/schema': {},
-    '@apostrophecms/attachment': {},
     '@apostrophecms/asset': {},
     '@apostrophecms/launder': {},
     '@apostrophecms/http': {},
     '@apostrophecms/i18n': {},
+    '@apostrophecms/attachment': {},
```

Comment:
This change has been reversed here, it was my first try at resolving the problem:

https://github.com/apostrophecms/apostrophe/commit/552dd6cd142438b3056ea504b5a7d566b1d11c88

The event-based change below in this PR is the keeper.